FEM/fem_ele.h

```diff
@@ -170,18 +217,34 @@ class CElement
 	int Order;
 	size_t ele_dim; // Dimension of element
 	size_t dim; // Dimension of real dimension
+	size_t dim_grad; // Gradient dimension
```

Comment:
thanks. can you rename those variables, e.g. ele_local_dim and ele_global_dim? I don't get what "grad" means.